lib/cell/builder.rb

```diff
@@ -9,7 +9,7 @@ def self.included(base)
 
     module ClassMethods
       def build(*args)
-        build!(self, *args).new(*args) # Declarative::Builder#build!.
+        build!(self, *args, **{}).new(*args) # Declarative::Builder#build!.
```

Comment:
Not sure if this is an appropriate fix, but it seems to work!

Reply:
Are you guys using Cells' builder features, and if yes, what do your builders look like?

Reply:
No we are not using them. I just found this by running the tests with ruby 2.7 and the tests still pass for ruby 3 so I *think* this may work.